### DISCUSSION TITLE
Start Button has moved! How do I return it to lower left corner?

### DISCUSSION THREAD

**Discussion by BigNick1954** [Open-Shell/Open-Shell-Menu#1941] [Q&A] [answered]
For some reason, the start button on Open Shell Menu 4.4.191 has suddenly moved to the middle os the taskbar.
How do I get it to move back? I can't find any setting to do this.
Thanks in advance.
BTW - I'm not a programmer!!

**Comment by ge0rdi** [comment, marked as answer]
See #1252 for discussion about Start button placement.

**Reply by BigNick1954** [reply]
So I simply go back to 180 and never update then?

**Comment by BigNick1954** [comment]
I've solved all my problems now!
Gone back to Classic Shell. Sorted.
